arangod/Aql/QuerySnippet.cpp

```diff
@@ -403,11 +403,14 @@ ResultT<std::unordered_map<ExecutionNode*, std::set<ShardID>>> QuerySnippet::pre
           // OneShard case) because local graphs (on DBServers) only ever occur
           // in either OneShard or SatelliteGraphs.
```

Comment:
Comment should be updated to reflect that disjoint smart graphs can occur here too.